liqd_product/config/settings/base.py

```diff
@@ -10,6 +10,23 @@
 # Application definition
 
 INSTALLED_APPS = (
+
+    'wagtail.contrib.settings',
+    'wagtail.wagtailforms',
+    'wagtail.wagtailredirects',
+    'wagtail.wagtailembeds',
+    'wagtail.wagtailsites',
+    'wagtail.wagtailusers',
+    'wagtail.wagtailsnippets',
+    'wagtail.wagtaildocs',
+    'wagtail.wagtailimages',
+    'wagtail.wagtailsearch',
+    'wagtail.wagtailadmin',
+    'wagtail.wagtailcore',
+    'wagtail.contrib.wagtailstyleguide',
+    'modelcluster',
+    'taggit',
```

Comment:
`liqd_product.apps.django_overwrites.apps.Config` has to be the first app that is loaded (first in list) because it overwrites/monkeypatches djangos reverse function. 
i'll add a comment to the following line to make this more obvious. 

i'd propose to make another block `# Wagtail cms components` for the wagtail core apps and our cms apps above the `# General adhocracy 4 components` block